docs/integrations/supabase.md

```diff
@@ -0,0 +1,137 @@
+
+![A screenshot of a supabase smtp settings.](./supabase-smtp-settings.webp)
+
+After entering these details, save the configuration.
+
+Keep in mind that supabase's interval and rate limit settings will still apply when sending emails through bluefox.email. Additionally, your email sending will be subject to your project's sending rate limit per second, so ensure your configuration stays within this limit to avoid delivery issues
+
+## Creating a Transactional Email
+
+To enable supabase to send authentication emails through bluefox.email, you need to create ``transactional email`` for each email type. At a minimum ``Confirm Signup`` and ``Magic Link`` must be set up for proper integration.
+
+**Supabase email types:**
+- Confirm Signup
+- Magic Link
+- Change Email Address
+- Password Reset
+- Reauthentication
+- User Invitation
+
+To create a **transactional email**, navigate to your **bluefox.email** project and [create a new transactional email](/docs/projects/transactional-emails#creating-a-transactional-email). This email will be used to send authentication-related messages based on supabase triggers.
+
+Inside the email template, use **merge tags** to dynamically insert content from supabase. For example, in a Confirm Signup email, link the ``{ { confirmation_url } }`` merge tag in your email template to ``{ { .ConfirmationURL } }``from supabase.
+
+The following merge tags provided by supabase allow you to personalize and structure the email content dynamically:
+
+  ```plaintext
+  {{ .ConfirmationURL }} - URL to confirm the e-mail address for the new account
+  {{ .Token }} - The 6-digit numeric email OTP
+  {{ .TokenHash }} - The hashed token used in the URL
+  {{ .SiteURL }} - The URL of the site
+  {{ .Email }} - The user's email address
+  {{ .Data }} - The user's `user_metadata`
+  {{ .RedirectTo }} – The URL of `emailRedirectTo` passed in options
+  ```
+
+Once your template is set up, save the transactional email. In the next step, you’ll configure supabase to use this template by mapping the JSON data structure correctly.
+
+
+
+## Configuring the Transactional Email in supabase
+
+After setting up your transactional email, you need to configure supabase to use it:
+
+1. In **bluefox.email**, navigate to the **code guide** section of your transactional email and locate the JSON body and copy it.
+
+![A screenshot of a bluefox.email code guide json body.](./transactional-email-code-guide-json-body.webp) 
+
+2. In **supabase**, go to **Authentication > Email Templates** and select the email type to configure.
+3. Paste the copied JSON inside the **Email Source** field and modify the `"data"` object to ensure it passes the correct supabase variables.
+
+![A screenshot of a bluefox.email code guide json body.](./supabase-email-template.webp) 
+
+Example JSON:
+  ```json
+   {
+     "email": "{{ .Email }}",
+     "transactionalId": "YOUR_TRANSACTIONAL_EMAIL_ID",
+     "data": {
+       "confirmation_link": "{{ .ConfirmationURL }}"
```

Comment:
Do we use this kind of snake_case variables in our templates?

I would use a property name here that we already regularly use... and I would mention to the user that they can use any variable, but they will need to change the merge tag on their button within bluefox when they opened the transactional email with the editor.

Reply:
Also, should not we include an example for every supabase email?